packages/ui/src/theme/css/component/inAppMessaging/messageLayout.scss

```diff
@@ -41,18 +41,22 @@
   }
   &__content {
     flex-grow: 1;
+    overflow: hidden;
     &--horizontal {
       flex-direction: row;
     }
     &--vertical {
       flex-direction: column;
       justify-content: center;
-      overflow-y: auto;
     }
   }
   &__header {
     font-size: var(--amplify-components-inappmessaging-header-font-size);
     font-weight: var(--amplify-components-inappmessaging-header-font-weight);
+    min-height: calc(
```

Comment:
I  think if we put a `flex-shrink: 0` on this header it will prevent it from vertically flexing down to nothing-ness and then we don't need this min-height calc?

Reply:
Excellent callout, thank you